### PR TITLE
Add sort command to sort the displayed list

### LINKED ISSUE #100: Add sort command to sort the displayed list

### PR DESCRIPTION
**Pull request by DesSnowy**
The `sort` command enables user to sort the displayed list based on `name`, `github` and `telegram`.
The user needs to specifty the order to be sorted by (asc or desc).

closes #100

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/SortCommand.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import seedu.address.model.person.comparator.PersonComparator;
+
+/**
+ * Sorts the current filtered list based on the specified sort type.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts the current list based on the specified field.\n"
```
Comment (currynia) [inline src/main/java/seedu/address/logic/commands/SortCommand.java:18]: Would it be better to write that the fields can be `github`, `name` or `telegram` and that `order` can be `asc` or `desc`?

**Review by currynia — APPROVED** [review]
Is it intended that the fields are sorted by their decimal values e.g.  j > Z. If yes then everything lgtm.

File: src/main/java/seedu/address/logic/commands/SortCommand.java (hunk 1 above)
Comment (SwaminathanViswa) [reply src/main/java/seedu/address/logic/commands/SortCommand.java:18]: Yup I agree with this. A brief message in the `MESSAGE_USAGE` would make it clearer for the reader and the rest can be in the User Guide.

Comment (DesSnowy) [reply src/main/java/seedu/address/logic/commands/SortCommand.java:18]: Right. I will add that in and change the comparsion as well.

**Pull request merged**